client/components/update-post-status/index.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import UpdateTemplate from './update-template';
+import PostActions from 'lib/posts/actions';
+import analytics from 'lib/analytics';
+
+// TODO(mcsf): use i18n#localize?
+const strings = {
```

Comment:
While we do a hard refresh when a user changes language settings, I think this is safer if this was translated in a lifecycle event, or